src/reactor.h

```diff
@@ -0,0 +1,242 @@
+    "units": "kg", \
+    "uilabel": "Reserve Inventory" \
+  }
+  double reserve_inventory;
+
+  #pragma cyclus var { \
+    "doc": "Tritium core inventory required to start reactor", \
+    "tooltip": "Tritium core inventory required to start reactor", \
+    "units": "kg", \
+    "uilabel": "Start-up Inventory" \
+  }
+  double startup_inventory;
+
+  #pragma cyclus var { \
+    "default": 'fill', \
+    "doc": "Method of refueling the reactor", \
+    "tooltip": "Options: 'schedule' or 'fill'", \
+    "uilabel": "Refuel Mode" \
+  }
+  std::string refuel_mode;
+
+  #pragma cyclus var { \
+    "default": 0.1, \
+    "doc": "Quantity of fuel which reactor tries to purchase", \
+    "tooltip": "Defaults to 100g/purchase", \
+    "units": "kg", \
+    "uilabel": "Buy quantity" \
+  }
+  double buy_quantity;
+
+ #pragma cyclus var { \
+    "default": 1, \
+    "doc": "Frequency which reactor tries to purchase new fuel", \
+    "tooltip": "Reactor is active for 1 timestep, then dormant for buy_frequency-1 timesteps", \
+    "units": "Timesteps", \
+    "uilabel": "Buy frequency" \
+  }
+  double buy_frequency;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string fuel_incommod;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string he3_outcommod;
+
+  #pragma cyclus var { \
+    "default": 1000.0, \
+    "doc": "Initial mass of full blanket material", \
+    "tooltip": "Only blanket material mass, not structural mass", \
+    "uilabel": "Initial Mass of Blanket" \
+  }
+  double blanket_size;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_incommod;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_inrecipe;
+
+  #pragma cyclus var { \
+    "default": 0.3, \
+    "doc": "Li-6 enrichment requirement percent as a decimal", \
+    "tooltip": "Defaults to 30% (0.3)", \
+    "units": "%", \
+    "uilabel": "Minimum Blanket Enrichment" \
+  }
+  double min_blanket_enrichment;
+
+
+  //WARNING: The default on this is completely arbitrary!
+  #pragma cyclus var { \
+    "default": 0.05, \
+    "doc": "Percent of blanket that gets recycled every timestep", \
+    "tooltip": "Defaults to 5% (0.05), must be between 0 and 15%", \
+    "units": "%", \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  double blanket_turnover_rate;
+
+  bool operational = true; 
+  bool core_loaded = false;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium kept in the core"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_core;
```

Comment:
It's not clear that this needs to be a `ResBuf` since its completely internal and always a single material

Reply:
Addressed. This got deleted in favor of the new way of doing things as per Matthew's comment about making the code match Abdou 2021 better.